src/pages/dashboard/panels/ImportBreakdown.tsx

```diff
@@ -39,7 +39,8 @@ function ImportBreakdown() {
   };
 
   return (
-    <ShokoPanel title="Import Breakdown" options={renderOptions()} isFetching={!(activeTab === 'unrecognized' ? hasFetchedUnrecognized : hasFetchedRecents)}>
+    // <ShokoPanel title="Unrecognized" options={renderOptions()} isFetching={!(activeTab === 'unrecognized' ? hasFetchedUnrecognized : hasFetchedRecents)}>
```

Comment:
you have to delete the function and you have to delete the unused tab too and all the code related to tab switching, so maybe just don't comment it for now and I will remove it

Reply:
Yeah, I didn't want to delete it just in case so just commented it out.